tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_unsuccessful_file_modification/rule_audit_rules_unsuccessful_file_modification_open_rule_order/unordered.fail.sh

```diff
@@ -3,4 +3,8 @@
 # profiles = xccdf_org.ssgproject.content_profile_ospp
 # remediation = none
 
-cp ./unordered.rules /etc/audit/rules.d/
+# Puts rule without filter first
```

Comment:
I believe this needs more explaining for anybody who is not deeply involved with the problem. Expand the comment a bit?

Reply:
@dahaic I have added clarification in rule prose, and test setup.